packages/xrpl/test/client/autofill.test.ts

```diff
@@ -8,18 +8,26 @@ import {
 } from 'xrpl-local'
 
 import rippled from '../fixtures/rippled'
-import { setupClient, teardownClient } from '../setupClient'
+import {
+  setupClient,
+  teardownClient,
+  type XrplTestContext,
+} from '../setupClient'
 import { assertRejects } from '../testUtils'
 
 const Fee = '10'
 const Sequence = 1432
 const LastLedgerSequence = 2908734
 
 describe('client.autofill', function () {
-  beforeEach(setupClient)
-  afterEach(teardownClient)
+  let testContext: XrplTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient()
+  })
+  afterEach(async () => teardownClient(testContext))
 
-  it('should not autofill if fields are present', async function () {
+  it('should not autofill if fields are present', async () => {
```

Comment:
Why the change in this syntax everywhere?

Reply:
I actually tried to revert this. The goal was to maintain a consistent pattern, as we have mixed styles everywhere

Reply:
If possible, I'd prefer to fix this in a separate PR (should be quick enough to switch back with a simple global ctrl-f-replace).

Reply:
It wasn't actually, I tried doing it with a regex but what ended up happening is changing Style A to Style B or changing Style B to Style A 🤷

Reply:
Let me know if we can resolve and worry about it later

Reply:
Shouldn't it just be `async () => ` to `async function ()`, with another ctrl-f-replace for the version with added line breaks?